service/controller/clusterapi/v29/adapter/adapter.go

```diff
@@ -48,6 +48,7 @@ type Config struct {
 	StackState                      StackState
 	TenantClusterAccountID          string
 	TenantClusterKMSKeyARN          string
+	TenantClusterAvailabilityZones  []string
```

Comment:
This value comes from `controllercontext` that has all distinct cluster AZs (master AZ + AZs from all `MachineDeployments`).